README.md

```diff
@@ -1,22 +1,26 @@
 # DesignAids
 
 Requirements:
------------------------------
-    1) Install LaTex
+----------------------------
+    1) Install LaTeX
     2) Install SageMath
-    3) Configure LaTex with Sagemath
+    3) Configure LaTeX with Sagemath
```

Comment:
Here you spelled it "Sagemath" and on line 17, it's "SageMath". Have some consistency.